gui/src/main/scala/co/uproot/abandon/RegReport.scala

```diff
@@ -30,34 +36,49 @@ object RegUIReport extends UIReport {
     new TreeView(reportRoot) {
       styleClass += styleClassName
       onKeyTyped = { e: KeyEvent =>
-        if (e.character equals "\r") {
-          val selectedItemOpt = selectionModel().getSelectedItems().headOption
-          selectedItemOpt foreach { selectedItem =>
-            // println(e)
-            val txStage = new Stage() {
-              scene = new Scene(800, 600) {
-                root = new ScrollPane {
-                  content = TxnUIReport.mkTxnView(getNestedTxns(selectedItem))
-                }
-                stylesheets += "default_theme.css"
-              }
-              initModality(Modality.ApplicationModal)
-              title = "Transactions"
+        if (e.character.equals("\r") && !selectionModel().isEmpty) {
+          // println(e)
+          selectedItem = selectionModel().getSelectedItem
+          // show all transactions of the parent of the selected item if the latter has any transactions
+          // this will allow for an update of the transaction view when an input file changes
+          if (selectedItem.getValue.txns.nonEmpty) selectedItem = selectedItem.getParent
```

Comment:
When you change to parent, the transaction view includes more transactions than the selected one.

Try this example:
* In "All" tab, select "Assets" and press Enter
* In the register report for "Assets", select "2013 / Jan" / "Assets:Current:TDS" and press Enter.
* Originally this showed transactions only for "Assets:Current:TDS", but now it will show all transactions in "2013 / Jan"